### ISSUE TITLE
Peer transit gateways

### ISSUE THREAD

**Issue by seanprivett** [ministryofjustice/modernisation-platform#888]
## User Story

As a… MOJO and DOM1 user
I need/want/expect to connect to an application on the Mod Platform
So that I can prove connectivity from my device

## Value

Early identification of gaps and mitigation of risk relating to network connectivity for end users

## Questions / Assumptions

MOJO peeps (Andy Cohen) to share their pre-production transit gateway with us before we start this

## Definition of done

- [ ] all to be done in sandbox account
- [ ] #889
- [ ] route tables set up
- [ ] no conflicting CIDR ranges
- [ ] update nacl rules
- [ ] test app endpoint can be accessed from DOM1 device
- [ ] test app endpoint can be accessed from MOJO device
- [ ] readme has been updated
- [ ] user docs have been updated
- [ ] another team member has reviewed
- [ ] tests are green

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by davidkelliott** [comment]
We are putting this on hold until there is a solid use case.  #889 proved one possible way of connecting the transit gateways, but there are multiple ways of doing this depending on the scenario.  We will look at this again when we need to do it.
